containers/elixir-phoenix-postgres/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Update the VARIANT arg in devcontainer.json to pick a Ruby version: 2, 2.7, 2.6, 2.5
```

Comment:
You should update this comment to reference Elixr versions rather than Ruby.  You would also need to update this in your `docker-compose.yml` file rather than devcontainer.json. (See other comment on this one.)

Reply:
Good call. This is what happens when you get something working late and open a PR. ;)

I'll get this all fixed up shortly.